pkg/api/v1alpha1/cluster_types.go

```diff
@@ -135,7 +135,8 @@ type ControlPlaneConfiguration struct {
 	// Count defines the number of desired control plane nodes. Defaults to 1.
 	Count int `json:"count,omitempty"`
 	// Endpoint defines the host ip and port to use for the control plane.
-	Endpoint *Endpoint `json:"endpoint,omitempty"`
+	Endpoint       *Endpoint `json:"endpoint,omitempty"`
+	ExtraEndpoints []string  `json:"extraEndpoints,omitempty"`
```

Comment:
Can you explain this addition as we would need to support this for other providers then, unless we will have specific validation for this based on this provider?

Reply:
Our customer has a usecase where they inject their own load balancer while the cluster is being created, as explained in https://quip-amazon.com/TwUbAhygIjuP/Expose-extra-SANs-attribute-for-Control-Plane-Load-Balancing. 

I see your point about needing to support this for other providers, however. @vignesh-goutham do you have any inputs about whether this is still needed and/or how we should address it moving forward?

Reply:
Let's call this `certSANs` to keep that consistent with kubeadm configuration

Reply:
Discussed offline - decided that this was not a blocker. If we want to move forward and add it, then add some validation for other providers like https://github.com/aws/eks-anywhere/blob/bfaa4d37237cdc3c8aeef58545a9bec212fc0a55/pkg/providers/docker/docker.go#L99, and also create an issue to add support for other providers

Reply:
Just to clarify, are we keeping this or removing it?

Reply:
Looks like we are removing this, but I am fine with adding it as `certSANs` and having similar validations too.

Reply:
Will re-add

Reply:
I'm confused
Do we need it?
If we don't, I vote to remove
If we do, then I think we should support it for all providers. But we should be aware that this increases the scope and adds more work on the testing side